### ISSUE TITLE
Manual drag and drop install of Thumbnails in RA 1.9.1+ disabled?

### ISSUE THREAD

**Issue by SamwiseOfCat** [libretro/RetroArch#12502]
**Description**

Odd error ive found on versions of RA after 1.9.0. I cant use F5 and then drag and drop thumbnails onto files in the playlists, which I could in 1.9.0. Is fixable by downgrading back to 1.9.0 (manual install via drag and drop from unpacked ZIP). 

**Expected Behavior**

Ability to drag and drop jpg files into Thumbnail window manually via F5 desktop menu.

**Actual Behaviour**

"Copy" dialogue appears when dragging jpg to Thumbnail window but jpg does not drop, thumbnails do not appear in RA 1.9.1+ even after a restart

**Hotfix**

Downgrade RA to 1.9.0 via manual file copy/overwrite

**Affected Versions**

RA 1.9.1+

**Environment information**

OS is Windows 10 20H2, Build 19042.985

**Comment by bslenul** [comment]
Yeah .jpg support is broken in Desktop Menu since 1.9.1, see #12238

**Comment by SamwiseOfCat** [comment]
> 
> 
> Yeah .jpg support is broken in Desktop Menu since 1.9.1, see #12238

Apologies - I did a previous issue search but missed your ticket. Yes - same issue it appears...
